Src/BootCamp.Chapter/Lesson3.cs

```diff
@@ -0,0 +1,93 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace BootCamp.Chapter
+{
+    class Lesson3
+    {
+        public static void Demo()
+        {
+            bool calculateAgain = true;
+
+            while (calculateAgain)
+            {
+                Console.WriteLine("BMI Calculator");
+
+                string name = getString("Enter your name: ");
+                int age = getInt("Enter your age: ");
+                float weight = getFloat("Enter your weight(kg): ");
+                float height = getFloat("Enter your height(cm): ");
+
+                Console.WriteLine(name + " is " + age + " years old, his weight is " + weight + " kg and his height is " + height + " cm.");
+
+                float BMI = calculateBMI(weight, height / 100);
+
+                Console.WriteLine("Your BMI is " + BMI);
+
+                if (BMI < 18.5)
+                {
+                    Console.WriteLine("You are Underweight.");
+                }
+                else if (BMI >= 18.5 && BMI < 25)
+                {
+                    Console.WriteLine("You have a Normal Weight.");
+                }
+                else if (BMI >= 25 && BMI < 30)
+                {
+                    Console.WriteLine("You are Overweight.");
+                }
+                else if (BMI >= 30)
+                {
+                    Console.WriteLine("You are Obese.");
+                }
+                else
+                {
+                    Console.WriteLine("Something went wrong. Please try again.");
+                }
+
+                Console.WriteLine("Do you want to calculate another persoon's BMI? (y/n)");
+                string continueTheFight = Console.ReadLine();
+                while (continueTheFight != "y" && continueTheFight != "n")
+                {
+                    Console.WriteLine("That is not a valid response.");
+                    Console.WriteLine("Do you want to calculate another persoon's BMI? (y/n)");
+                    continueTheFight = Console.ReadLine();
+                }
+
+                if (continueTheFight == "n")
+                {
+                    calculateAgain = false;
+                }
+
+            }
+
+            Console.WriteLine("Exiting...");
+                        
+        }
+                
+        public static string getString(string message)
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Rename method 'getString' to match pascal case naming rules, consider using 'GetString'.](https://app.codacy.com/manual/Almantask/CSharp-From-Zero-To-Hero/pullRequest?prid=4863604)